Rubberduck.VBEEditor/Events/EventArgs/KeyPressEventArgs.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Windows.Forms;
+
+namespace Rubberduck.VBEditor.Events
+{
+    public class KeyPressEventArgs
+    {
+        public KeyPressEventArgs(IntPtr hwnd, IntPtr wParam, IntPtr lParam, char character = default)
```

Comment:
you should maybe explain why we are reinventing the WindowsApi.KeyPressEventArgs here

Reply:
Not mine, just moved it there. It passes the arguments from the window proc though. Commented.

Reply:
That was me. This was introduced for AC to pick up both WM_CHAR and WM_KEYPRESS - the former gives us the typed characters, the latter a WinForms.Keys enum for non-characters, e.g. BACKSPACE. There's no reliable or obvious mapping between the WM_CHAR lparam and WinForms.Keys, so I folded both into this one.